homework-submission/react-hyf/react-hyf-client/src/components/week3/component/UserInfo.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+// import './Homework3.css';
+
+function UserInfo({ randomUser }) {
+  return (
+    <div className="card">
+      <img className="card-photo mb-4" src={randomUser.photo} alt="" />
+      <h3 className="text-center">
+        {randomUser.name} -{randomUser.surname}
```

Comment:
Good moment to deconstruct the properties of `randomUser` so that you don't have to repeat the variable all the time, resulting in more readable code